keygen/tests/web.rs

```diff
@@ -0,0 +1,47 @@
+//! Test suite for the Web and headless browsers.
```

Comment:
Good.
I wonder if it's possible to easily add Javascript tests for the node.js package as well and have them run by the CI. Not subject for PR, though.

Reply:
Well, if you mean javascript tests written in javascript, then the only thing I'll have to figure out is how to do is the CI part, as I did write most of the tests in `js-chain-libs` in javascript ([like this](https://github.com/input-output-hk/js-chain-libs/blob/master/tests/test_make_transaction.js)).